index.html

```diff
@@ -1282,6 +1241,7 @@ <h2>Terms of Use</h2>
 The group is currently exploring a variety of ways of expressing the terms of
 use associated with a Verifiable Credential, namely, the
 <a href="http://w3c.github.io/poe/model/">Open Digital Rights Language</a>.
+A related initiative is the one at customercommons.org.
```

Comment:
Change customercommons.org to &lt;a href="https:// customercommons.org">Customer Commons&lt;/a>

Reply:
will revise